components/email/src/Email.svelte

```diff
@@ -302,12 +304,7 @@
     }
   }
 
-  $: if (
-    !_this.thread &&
-    _this.clean_conversation &&
-    ((activeThread && !activeThread.messages.some((m) => m.conversation)) ||
-      (_this.message && !_this.message.conversation))
-  ) {
+  $: if (_this.clean_conversation) {
```

Comment:
This will result in us trying to fetch clean conversation even when a thread is being passed manually. We should only fetch if a thread _id or a message_id is passed.